google-cloud-core/pom.xml

```diff
@@ -27,6 +27,10 @@
       <groupId>com.google.api</groupId>
       <artifactId>gax</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I may be in a minority here, but I do not like autovalue. I've seen nothing but a constant stream of low-level maintenance pain from it.

Reply:
It was recommended by @chingor13, I can revert if it's really something we want to stay away from.